pages/common/docker-images.md

```diff
@@ -0,0 +1,20 @@
+# docker images
+
+> Manage Docker images.
+> More information: <https://docs.docker.com/engine/reference/commandline/images/>.
+
+- List all Docker images:
+
+`docker images`
+
+- List all Docker images with intermediate included:
```

Comment:
intermediates ? Maybe better is - List all Docker images including intermediates.